docs/source/monitoring_stack.rst

```diff
@@ -65,17 +65,17 @@ Install Scylla Monitoring
 
 .. code-block:: sh
 
-   wget https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-3.6.0.tar.gz
-   tar -xvf scylla-monitoring-3.6.0.tar.gz
-   cd scylla-monitoring-scylla-monitoring-3.6.0
+   wget https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-|version|.tar.gz
+   tar -xvf scylla-monitoring-|version|.tar.gz
+   cd scylla-monitoring-scylla-monitoring-|version|
 
 As an alternative, you can clone and use the Git repository directly.
 
 .. code-block:: sh
 
    git clone https://github.com/scylladb/scylla-monitoring.git
    cd scylla-monitoring
-   git checkout branch-3.6
```

Comment:
There is a problem here, when you check out, it's ``branch-3.6``

Reply:
I will address this in a different PR if needed.